city_scrapers/spiders/chi_ssa_69.py

```diff
@@ -0,0 +1,97 @@
+import json
+import re
+from datetime import datetime, timedelta
+
+from city_scrapers_core.constants import COMMISSION, COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa69Spider(CityScrapersSpider):
+    name = "chi_ssa_69"
+    agency = "Chicago Special Service Area #69 95th & Ashland Avenue"
+    timezone = "America/Chicago"
+
+    @property
+    def start_urls(self):
+        today = datetime.now()
+        last_week = today - timedelta(days=7)
+        in_two_months = today + timedelta(days=60)
+        chicago_timezone = "America/Chicago"
+
+        return [(
+            "https://www.googleapis.com/calendar/v3/calendars/gagdcchicago%40gmail.com/events"
+            "?calendarId=gagdcchicago@gmail.com&singleEvents=true&timeZone={}&"
+            "sanitizeHtml=true&timeMin={}T00:00:00{}&timeMax={}T00:00:00{}&"
+            "key=AIzaSyC-KzxSLmmZitsCVv2DeueeUxoVwP0raVk"
+        ).format(
+            chicago_timezone, last_week.strftime("%Y-%m-%d"), chicago_timezone,
+            in_two_months.strftime("%Y-%m-%d"), chicago_timezone
+        )]
+
+    def parse(self, response):
+        data = json.loads(response.text)
+
+        for item in data["items"]:
+            title = self._parse_title(item)
+            location = self._parse_location(item)
+            if not location:
```

Comment:
Might be quicker to just have `if not location or title` here

Reply:
I had to think about this for a while.
I think it needs to be:
if not (location and title):
    continue

Both location and title need to be present in order to process the meeting (continue bypasses the processing)

Let me know if I'm wrong.

Reply:
Ah yep, you're right that's my mistake